storage/database/dynamodb_test.go

```diff
@@ -113,6 +133,80 @@ func testDynamoBatch_WriteLargeData(t *testing.T) {
 	}
 }
 
+// testDynamoBatch_WriteMutliTables checks there is no error when working with more than on tables.
+// This also checks if shared workers works as expected.
+func testDynamoBatch_WriteMutliTables(t *testing.T) {
+	// this test might end with Crit, enableLog to find out the log
+	//enableLog()
+
+	// creat DynamoDB1
+	dynamo, err := NewDynamoDB(GetDefaultDynamoDBConfig())
+	defer dynamo.deletedDB()
```

Comment:
Why is it `deleted` instead of `delete`?

Reply:
Thank you. Review reflected :)